node.gyp

```diff
@@ -236,6 +236,7 @@
         'NODE_WANT_INTERNALS=1',
         # Warn when using deprecated V8 APIs.
         'V8_DEPRECATION_WARNINGS=1',
+        'V8_EMBEDDER_STRING=".node.0"'
```

Comment:
common.gypi or config.gypi?  With common.gypi it's kind of gnarly that it ends up being defined everywhere, including add-ons.